docs/source/api_reference/forecasting.rst

```diff
@@ -417,6 +417,7 @@ Generic framework adapters that expose other frameworks in the ``sktime`` interf
     :template: class.rst
 
     HCrystalBallAdapter
+    AutoTSAdapter
```

Comment:
this is not the right section, since the adaptation is private, not public.

A public adapter, for this section, is where both "ends" are composable like `sklearn`. However, `autots` is primarliy monolithic with some light (but non-composable/inconsistent) internal API, an we are adapting only the main entry point at the moment.

None of the other sections fit either, so what I would do is open a new taxonomic class "All-in-one autoML models" or similar - what do you think? Any opinions, @yarnabrina?

Reply:
No opinion, but I think @MBristle intends to make AutoTSAdapter public? This is designed as a concrete estimator I think, not something like adapters for statsmodels or statsforecast.

Reply:
The interface is the default public interface of the AutoTS library. So yes, I think it is already a concrete implementation. As mentioned by @fkiraly the AutoTS has its own forecasters implemented and uses them for AutoML (Interface is not compatible with sktime). So yes, it might be a new category something like "AutoML Forecasting".  Single Models of the AutoTS can be used by choosing the right configuration (model_name parameter).

Reply:
Re adapter, I am differentiating framework adapter from single-use-adapter. The `adapters` folder only has framework adapters, i.e., adapters between two frameworks, both following the strategy oop pattern

A framework adapter is sth like a US-to-EU power outlet adapter, which allows you to use any US electric device with any EU power outlet, this assumes a large collection of EU power outlets, EU electric devices, US electric devices.

In python, that would be sth like `sk_obj = Adapter(third_party_obj)`, where `third_part_obj` can be an instance of 100 different classes `third_party_cls1`, `third_party_cls2`, etc.

However, the `autots` interface targets only a single class from `autots` - it is an adapter but not a framework adapter.

A framework adapter would be sth allowing to use *all forecasters from `autots` as `sktime` forecasters* (not sure if that is feasible, e.g., if interfaces are comparable and homogenous within `autots`), without having to go through the "main class".

Maybe I'm nitpicking too much, either way it would be more convenient for the user to present the class in `forecasting.autots`.